pkg/lzma/lzma.go

```diff
@@ -0,0 +1,46 @@
+// Package lzma implements reading and writing of LZMA compressed files.
+//
+// This package is specifically designed for the LZMA format used popular UEFI
+// implementations. It requires the `lzma` and `unlzma` programs to be
+// installed and on the path.
+package lzma
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+const (
+	lzmaEncode = "lzma"
+	lzmaDecode = "unlzma"
+)
+
+// Decode decodes a byte slice of LZMA data.
+func Decode(encodedData []byte) ([]byte, error) {
+	cmd := exec.Command(lzmaDecode)
+	cmd.Stdin = bytes.NewBuffer(encodedData)
+	cmd.Stderr = os.Stderr
+	return cmd.Output()
+}
+
+// Encode encodes a byte slice with LZMA.
+func Encode(decodedData []byte) ([]byte, error) {
+	return EncodeLevel(decodedData, 6)
```

Comment:
why default to 6? is that the default for the regular lzma command? I thought it was 7

Reply:
This man pages says the default is -6: https://linux.die.net/man/1/lzma

Reply:
I see